actions/upgrade-actions.py

```diff
@@ -62,7 +62,9 @@ def _dry_run(containerd, gpu):
 
     result = {}
     for name, pkg in search.items():
-        available, installed = map(PkgVersion, (pkg.version, pkg.current_ver.ver_str))
+        current_ver = pkg.current_ver.ver_str if pkg.current_ver else "0.not-installed.0"
+        # there could be a package not install with a current version, assume it needs updating?
+        available, installed = map(PkgVersion, (pkg.version, current_ver))
```

Comment:
setting up the manual test required me to put the charm in a situation where `pkg.current_ver` was `None`.  

so I've added this in here so that we can still do a version comparison below

```bash
❯ dpkg --compare-versions 0.0.0 gt 0.not-installed.0; echo $?
1
```
shows that 0.0.0 > 0.not-installed.0

Reply:
LGTM, maybe add unit tests for this `dpkg` bit?